aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -351,6 +351,11 @@ module AptosFramework::Account {
         TestCoin::register(&signer);
     }
 
+    public(script) fun create_and_fund_account(funder: &signer, account: address, amount: u64) {
+        create_account(account);
+        TestCoin::transfer(funder, account, amount);
```

Comment:
This would now need to become Coin::transfer<TestCoin>(funder, account, amount).

An alternative here in case it's hard to find the appropriate funder is to mint an account some tokens after creation. You should be able to make a transaction (to create account and then mint) as the core resources account in the test.

Reply:
Yeah I can do a script transaction as well.